README.md

```diff
@@ -567,18 +567,27 @@ If you need additional Caddy plugins, or need to use a specific version of Caddy
 The main difference from the instructions on the official image is that you must override `CMD` to have the container run using the `caddy docker-proxy` command provided by this plugin.
 
 ```Dockerfile
-ARG CADDY_VERSION=2.5.0
+ARG CADDY_VERSION=2.6.1
 FROM caddy:${CADDY_VERSION}-builder AS builder
 
 RUN xcaddy build \
-    --with github.com/lucaslorentz/caddy-docker-proxy/plugin \
+    --with github.com/lucaslorentz/caddy-docker-proxy/v2 \
     --with <additional-plugins>
 
 FROM caddy:${CADDY_VERSION}-alpine
 
+EXPOSE 80 443 2019
+
+ENV XDG_CONFIG_HOME /config
+ENV XDG_DATA_HOME /data
+
+RUN apk add -U --no-cache ca-certificates curl
```

Comment:
These are all unnecessary/redundant, the base image already sets these appropriately.